POUF1.md

```diff
@@ -0,0 +1,442 @@
+
+The Director repository identifies a vehicle at the start of the update process using a stored vehicle version manifest. The Director checks the vehicle version manifest as described in the standard, but does not make any additional checks. This POUF does not do dependency resolution as the reference implementation is not used in practice and so does not have conflicts.
+
+This POUF does not support sending vehicle version manifests as [diffs](https://uptane.github.io/uptane-standard/uptane-standard.html#construct_manifest_primary) as the reference implementation is not concerned about potential bandwidth issues.
+
+# Metadata
+This POUF supports some metadata features that are optional in the Uptane Standard, but are required for an implementation of this POUF. These features are included in the metadata definitions in {Formats}. The details of these features are:
+* The metadata version will increment on an update to the metadata.
+* Root metadata must support [mappings of roles to lists of URLs as described in TAP 5](https://uptane.github.io/uptane-standard/uptane-standard.html#root_meta).
+* Repository mapping metadata will be included using a map file. The format of this file is described in {Formats}. In the mapping metadata, files are expressed using an enumerated list that supports wildcards.
+* The OEM will provide unencrypted images to ECUs, hence the Director repository’s targets metadata will contain hashes of unencrypted images. Metadata about encrypted images may be included in the custom field of targets metadata as described in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images).
+* The custom field of targets metadata on the Director repository must support an [encrypted target field for encrypting images per ECU](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images). In addition, the custom field will contain a release counter and hardware id from both the Director and Image repositories. This POUF does not support the director adding a download URL to the custom field of targets metadata.
+* The custom field of targets metadata on the Director and Image repositories are not compared. This means that the must match functionality described in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#custom-metadata-about-images) is not supported. A client may choose to compare some fields of targets metadata, but these checks are not required by this POUF.
+* The Image repository’s targets metadata may include metadata about delegations. [Delegations may be any depth and may also be cyclic](https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta).
```

Comment:
Aren't these always true?

Reply:
I read this as a MAY in the standard (https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta), but it does seem like something that should always be supported.